transmart-rest-api/grails-app/services/org/transmartproject/rest/dataExport/ExportService.groovy

```diff
@@ -0,0 +1,170 @@
+package org.transmartproject.rest.dataExport
+
+import grails.transaction.Transactional
+import org.quartz.JobBuilder
+import org.quartz.JobDataMap
+import org.quartz.JobDetail
+import org.quartz.Scheduler
+import org.quartz.TriggerBuilder
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.stereotype.Component
+import org.transmartproject.core.exceptions.InvalidRequestException
+import org.transmartproject.db.job.AsyncJobCoreDb as AsyncJob
+import org.transmartproject.db.user.User
+
+import javax.transaction.NotSupportedException
+
+@Transactional
+@Component("restExportService")
+class ExportService {
```

Comment:
There is a lot of logic shared with `DataExportService` and `ExportAsyncJobService`. Let's have a look whether we could reduce the number of services.